internal/attribute/attribute_test.go

```diff
@@ -94,3 +94,13 @@ func TestSliceValue(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkSliceValue(b *testing.B) {
+	for n := 0; n < b.N; n++ {
+		val := StringSliceValue([]string{"a", "b", "c", "d"})
+
+		if val.([4]string) != [...]string{"a", "b", "c", "d"} {
+			b.Errorf("unexpected result")
+		}
```

Comment:
```suggestion
		_ = StringSliceValue([]string{"a", "b", "c", "d"})
```

In this benchmark, you're also running the value comparison. Benchmarks should run the method they assert.

Reply:
I was afraid it would get optimized away though. But I can change it

Reply:
I mean the value assertion should be a test, not a benchmark.

Reply:
Ok I removed the value comparison, and checked it has no impact. (the benchmark still really calls the function).

My point was that since the method called has no side effect it could be optimized away in a future go version with a smarter compiler.